docs/api.md

```diff
@@ -47,10 +47,28 @@ TODO
 TODO
 #### close()
 TODO
+
 ## Constants
-TODO
+### Registers
+You should use these when referencing registers. Under the hood, they all reference numbers that actually get passed to the Unicorn engine. See each architecture's respective documentation for more information on their registers.
+#### ARM64
+Formatted as `ARM64_REG_[register name]`.
+#### ARM
+Formatted as `ARM_REG_[register name]`.
+#### M68K
+Formatted as `M68K_REG_[register name]`.
+#### MIPS
+Formatted as `MIPS_REG_[register name]`.
+#### SPARC
+Formatted as `SPARC_REG_[register name]`.
+#### X86
+Formatted as `X86_REG_[register name]`.
+### X86 Instructions
+You should use these when referencing X86 instructions (should you ever do so rather than using Capstone.js or Keystone.js). Under the hood, they all reference numbers that actually get passed to the Unicorn engine. Formatted as `X86_INS_[instruction name]`. See the X86 documentation for more information on its instructions.
+### Unicorn Constants
+TODO (I'm almost sure this will be like **Helpers**, where it shouldn't be documented.)
```

Comment:
As you pointed out, it doesn't make sense to document these separately, as it would not offer significant advantage over "just reading the *unicorn-constants.js * ile"

\*__If__\* documented at all, we should do so in the relevant functions, e.g. `HOOK_[type]` constants can be described in the documentation for the `hook_add`` function.

```suggestion
```